app/assets/stylesheets/era_custom.scss

```diff
@@ -1,3 +1,9 @@
+@import 'avalon';
```

Comment:
Shouldn't need to import avalon here (considering your not using anything from it)? avalon and this file (era_custom) get imported from the application.scss file